bluegraph/backends/utils.py

```diff
@@ -0,0 +1,49 @@
+"""A set of utils for dispatching and using different processing backends."""
```

Comment:
I would call this factory ( this is to build) rather than dispatching  ( which bear the idea of routing)